master/jobmanager.go

```diff
@@ -24,18 +28,53 @@ type JobManager struct {
 
 	mu          sync.Mutex
 	jobMasters  map[model.ID]*model.Task
-	idAllocater autoid.JobIDAllocator
+	idAllocator autoid.JobIDAllocator
 	clients     *client.Manager
 	masterAddrs []string
 }
 
-func (j *JobManager) Start(ctx context.Context) error {
+func (j *JobManager) recover(ctx context.Context) error {
+	resp, err := j.MetaKV.Get(ctx, adapter.JobKeyAdapter.Path())
+	if err != nil {
+		return err
+	}
+	result := resp.(*clientv3.GetResponse)
+	for _, kv := range result.Kvs {
+		jobID, err := strconv.Atoi(string(kv.Key))
+		if err != nil {
+			return err
+		}
+		jobTask := &model.Task{
+			ID:   model.ID(jobID),
+			OpTp: model.JobMasterType,
+			Op:   kv.Value,
+			Cost: 1,
+		}
+		err = j.RestoreTask(ctx, jobTask)
```

Comment:
why the jobmanager should need restore task?

Reply:
job master is a kind of task in job manager